client/src/components/Public/EntitySearchAutocomplete/EntitySearchAutocomplete.js

```diff
@@ -102,12 +102,12 @@ const EntitySearchAutocomplete = ({
           )
         }}
       />
-      <InputGroupAddon addonType="append">
+      <InputGroupAddon title={FIND_ENTITY_TITLE} addonType="append">
         <Button className="addon-button" color="primary" onClick={findEntities}>
           <SearchIcon />
         </Button>
       </InputGroupAddon>
-      <InputGroupAddon addonType="append">
+      <InputGroupAddon title="Otvor modálne okno" addonType="append">
```

Comment:
So... Why is one of them a constant and the second one just string? :slightly_smiling_face: This is pure nitpicking, but please unify.

Reply:
I originally reused the placeholder text that was already a constant. Fixed